### PR TITLE
Update README.md and metadata.json for next releases

### PR DESCRIPTION
**Pull request by adityapatwardhan**
# PR Summary

Update the metadata.json and README.md for upcoming releases.

## PR Context

This PR updates the files for release v6.2.2 and v6.1.5

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,21 +57,21 @@ You can also download the PowerShell binary archives for Windows, macOS and Linu
 | Windows (arm) **Experimental**      | [32-bit][rl-winarm]/[64-bit][rl-winarm64]        | [32-bit][pv-winarm]/[64-bit][pv-winarm64]       | [Instructions][in-arm]                         |
```
Comment (PoshChan) [inline README.md]: @adityapatwardhan, your last commit had 15 failures in `PowerShell-CI-static-analysis`
(These are 5 of the failures)

Verify Markdown Links.Verify links in /home/vsts/work/1/s/README.md.https://github.com/PowerShell/PowerShell/releases/download/v6.2.2/PowerShell-6.2.2-win-x64.msi should work
```powershell
retry of URL failed with error: 
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 111
111:                                 throw "retry of URL failed with error: $($_.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/README.md.https://github.com/PowerShell/PowerShell/releases/download/v6.2.2/PowerShell-6.2.2-win-x86.msi should work
```powershell
retry of URL failed with error: 
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 111
111:                                 throw "retry of URL failed with error: $($_.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/README.md.https://github.com/PowerShell/PowerShell/releases/download/v6.2.2/powershell_6.2.2-1.ubuntu.18.04_amd64.deb should work
```powershell
retry of URL failed with error: 
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 111
111:                                 throw "retry of URL failed with error: $($_.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/README.md.https://github.com/PowerShell/PowerShell/releases/download/v6.2.2/powershell_6.2.2-1.ubuntu.16.04_amd64.deb should work
```powershell
retry of URL failed with error: 
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 111
111:                                 throw "retry of URL failed with error: $($_.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/README.md.https://github.com/PowerShell/PowerShell/releases/download/v6.2.2/powershell_6.2.2-1.debian.9_amd64.deb should work
```powershell
retry of URL failed with error: 
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 111
111:                                 throw "retry of URL failed with error: $($_.Message)"

```

**Review by TravisEz13 — COMMENTED** [review]
Looks good, but not ready to merged

**Comment by adityapatwardhan** [comment]
@PoshChan please retry static

**Comment by PoshChan** [comment]
@adityapatwardhan, successfully started retry of `PowerShell-CI-static-analysis`

**Review by TravisEz13 — APPROVED** [review]

**Comment by TravisEz13** [comment]
Let’s set the commit for the where the commit will be released.  We created this commit here, so it will be released in preview.3

**Pull request merged**